javaslang/src/main/java/javaslang/control/GwtIncompatible.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2013 Google Inc.
```

Comment:
The Apache License allows you to do just that; but I believe you should then add the Google copyright to your `LICENSE` file, or at least add a note that parts of the code have a Google copyright.

But actually you don't need to copy the file: GWT expects an annotation with a simple of `GwtIncompatible`, in any package, will ignore any annotation attribute, and will only process the annotation on classes, fields, constructors and methods, and that's it. So just create such an annotation in your project that fulfills that contract, and put your own license (and copyright) on it.